docs-src/stargate-develop/modules/develop/examples/docapi/library/_test.sh

```diff
@@ -132,8 +132,9 @@ echo "Put JSON schema for a particular collection"
 echo "Get all collections, to check that the 2 collections are made."
 ./curl-collection-get-all.sh.tmp|jq -r '.'
 # LIST JSON SCHEMA FOR A PARTICULAR COLLECTION
-echo "Get JSON schema for a particular collection"
-./curl-collection-get-json-schema.sh.tmp | jq -r '.'
+# LLP - EricB says this code was deleted
+#echo "Get JSON schema for a particular collection"
```

Comment:
This wasn't deleted! I just thought a note about the feature should be added, something like: `Note: Partial updates on documents are not allowed when a JSON schema is being used.`

Reply:
I did add a note about the JSON schema. And I commented out the wrong thing in the test.